fdbkubernetesmonitor/monitor.go

```diff
@@ -152,7 +156,14 @@ func StartMonitor(ctx context.Context, logger logr.Logger, configFile string, cu
 	// Add Prometheus support
 	mux.Handle("/metrics", promHandler)
 	go func() {
-		err := http.ListenAndServe(listenAddr, mux)
+		if promConfig.keyFile != "" || promConfig.certFile != "" {
```

Comment:
```suggestion
		if promConfig.keyFile != "" && promConfig.certFile != "" {
```

Based on https://pkg.go.dev/net/http#ListenAndServeTLS `Additionally, files containing a certificate and matching private key for the server must be provided` this method expects both the cert file and key file.

Reply:
oh this is more about catching the case where one is specified but not the other by accident.  ListenAndServeTLS should fail without both